doc/pages/architecture/060-triggers.md

```diff
@@ -106,8 +107,11 @@ This is the generic representation of a Data Trigger:
 - `device_connected`: triggered when a device connects to its transport.
 - `device_disconnected`: triggered when a device disconnects from its transport.
 
-`device_id` can be a specific Device ID or `*`, meaning the trigger will be installed for all
-devices in a Realm.
+`device_id` can be used to pass a specific Device ID to restrict the trigger to a single device.
+
+`group_name` can be used to restrict the trigger to all devices that are member of the group.
```

Comment:
It is unclear what happens when both of them are empty. You should mention that explicitly. Also, if the old `*` is retained, you should mention it at least.